apps/server/src/modules/learnroom/controller/dto/dashboard.response.ts

```diff
@@ -88,6 +88,11 @@ export class DashboardGridElementResponse {
 	})
 	isSynchronized: boolean;
 
+	@ApiProperty({
+		description: 'Is the course locked?',
```

Comment:
Differs quite a bit from the description in `CourseMetadataResponse` - "Indicates if the course is locked and cannot be accessed.". Maybe more verbose is better?